frontend/src/container/Trace/TraceTable/index.tsx

```diff
@@ -129,7 +129,12 @@ function TraceTable(): JSX.Element {
 		if (!Array.isArray(sort)) {
 			const { order = spansAggregateOrder } = sort;
 			if (props.current && props.pageSize) {
-				const spanOrder = order === 'ascend' ? 'ascending' : 'descending';
```

Comment:
make a util for this one don't write logic in the component itself

Reply:
Done